l10n/de.json

```diff
@@ -160,7 +160,11 @@
   "photos": {
     "translations": {
       "Photos": "Fotos",
-      "Edit album details": "Album umbenennen"
+      "Edit album details": "Album umbenennen",
+      "Rename album": "Album umbenennen",
+      "Download all files in album": "Album herunterladen",
+      "Download selected files": "Auswahl herunterladen",
+      "Remove selection from album": "Auswahl vom Album entfernen"
```

Comment:
do we add translations for nextcloud features?

Reply:
Translations now removed as discussed

Reply:
Removed